### PR TITLE
Dashboard import routes

### LINKED ISSUE #12328: Add import route and overlay to org dashboards

### PR DESCRIPTION
**Pull request by ebb-tide**
Closes #12328

### CODE REVIEW

File: ui/src/dashboards/components/dashboard_index/DashboardsIndex.tsx [hunk 1]
```diff
@@ -8,7 +8,6 @@ import DashboardsIndexContents from 'src/dashboards/components/dashboard_index/D
 import {Page} from 'src/pageLayout'
 import SearchWidget from 'src/shared/components/search_widget/SearchWidget'
 import AddResourceDropdown from 'src/shared/components/AddResourceDropdown'
-import ImportDashboardOverlay from 'src/dashboards/components/ImportDashboardOverlay'
```
Comment (ischolten) [inline ui/src/dashboards/components/dashboard_index/DashboardsIndex.tsx]: looks like theres both a ImportDashboardOverlay and DashboardImportOverlay. should one of these be deleted?

File: ui/src/dashboards/components/DashboardImportOverlay.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  orgs: Organization[]
+}
+
+interface OwnProps extends WithRouterProps {
+  params: {orgID: string}
+}
+
+type Props = OwnProps & StateProps & DispatchProps
+
+class ImportDashboardOverlay extends PureComponent<Props> {
```
Comment (ischolten) [inline ui/src/dashboards/components/DashboardImportOverlay.tsx]: should this be DashboardImportOverlay to match the file name

File: ui/src/organizations/containers/OrgDashboardsIndex.tsx [hunk 3]
```diff
@@ -43,50 +43,49 @@ type Props = WithRouterProps & RouterProps & DispatchProps & StateProps
...
+                        </SpinnerContainer>
+                      )}
+                    </GetOrgResources>
+                  </TabbedPageSection>
+                </Tabs.TabContents>
+              </Tabs>
+            </div>
+          </Page.Contents>
+        </Page>
+        {this.props.children}
```
Comment (OfTheDelmer) [inline ui/src/organizations/containers/OrgDashboardsIndex.tsx]: nice!

File: ui/src/dashboards/components/DashboardImportOverlay.tsx (hunk 2 above)
Comment (ebb-tide) [reply ui/src/dashboards/components/DashboardImportOverlay.tsx]: thank you !

**Pull request merged**